### ISSUE TITLE
[Suggestion] Adaptive Oak Item List

### ISSUE THREAD

**Issue by InQuognito** [pokeclicker/pokeclicker#652]
It would be nice to have a list of the items you have equipped, instead of the last one selected. This would allow you to view the progress of all equipped items, not just the last one, and also fix the weird tracking issue of having to click around the ones you already have equipped to view/upgrade/track the one you want.

**Comment by anapple96** [comment]
known issues

**Comment by InQuognito** [comment]
@anapple96 want me to close it?

**Comment by Ishadijcks** [comment]
It is good that when the time for an upgrade the item flashes to notify players.

However when the item is already equipped I need to un-equipped it first before I can upgrade.

It may be better if we can scroll through all the items inside the upgrade box.
So players no need to take off the equipped item and forget to equip it once again after it has been upgraded.
I once forget to re-equip the shiny charm when upgrade is done.......

**Comment by Ishadijcks** [comment]
Hmm I like the ideas but I'm not sure what the UI could look like. A list would require more space. We could overlay the xp progress bar over the image, but that would make upgrading a bit awkward.

**Comment by RedSparr0w** [comment]
Just add a modal for all the possible equips?
Or an expand for when you are changing, when minimized, only shows the equipped items.

**Comment by Ishadijcks** [comment]
Decent idea

**Comment by InQuognito** [comment]
@Ishadijcks You could compress the current UI a bit when multiple are equipped. For instance, instead of:
![image](https://user-images.githubusercontent.com/44120299/79485234-8178d800-7fda-11ea-879c-7807c52de3b8.png)

You could do:
| Magic Ball | Gives a bonus to your catchrate | progress bar | level | bonus |
| Amulet Coin | Gain more coins from battling | progress bar | level | bonus |

Or even remove the description:
| Magic ball | Progress Bar | Level | Bonus |
|Amulet Coin | Progress Bar | Level | Bonus |
